### PR TITLE
Add failOnStderr property to linting task

### PR DESCRIPTION
**Pull request by chelnak**
### Context
@defong pointed out that even though linters were failing the build was still passing.

This PR adds failOnStderr to the linting task so that the build will fail in the correct place

### Changes proposed in this pull request
Add failOnStderr propert to linting task

### Guidance to review
Does this fail builds as expected?

### CODE REVIEW

**Comment by misaka** [comment]
I see that the mechanism to run the tests has been moved into a script, but I don't see where `failOnStderr` is enabled or anything. Is that a change done in the GUI?

Pretty annoying that `govuk-lint-ruby` doesn't return an error, sounds like that should also be fixed upstream.

File: docker-build-az.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Build, test and push a custom docker image
+set -e
+
+if [ "$SYSTEM_DEBUG" = 'true' ];
```
Comment (misaka) [inline docker-build-az.sh]: Don't need the `;` here unless `true` is on the same line.

Comment (misaka) [reply docker-build-az.sh]: > Don't need the `;` here unless `true` is on the same line.

`then` not `true`

**Comment by chelnak** [comment]
PR is now redundant

**Pull request closed without merging**